website/src/pages/index.astro

```diff
@@ -1,88 +1,134 @@
 ---
 import BaseLayout from '../layouts/base/BaseLayout.astro';
 import DataInfo from '../layouts/base/footer/DataInfo.astro';
+import { BrandName } from '../layouts/base/header/Brand';
 import { organismConfig } from '../types/Organism';
 import { dataOrigins } from '../types/dataOrigins';
 import { Page } from '../types/pages';
 import { ServerSide } from '../views/serverSideRouting';
+
+/* eslint-disable @typescript-eslint/naming-convention */
+const iconMapping: { [key: string]: string } = {
+    'Sequencing efforts': 'mdi--test-tube',
+    'Single variant': 'mdi--magnify',
+    'Compare side-by-side': 'mdi--compare-horizontal',
+    'Browse data': 'mdi--database-outline',
+};
+/* eslint-enable */
 ---
 
 <style>
+    a {
+        @apply text-slate-500 hover:underline;
+    }
```

Comment:
This is repeated from `data.astro`. Should we maybe extract it to a separate component instead (Astro or React) use that when we want to have links?

Reply:
I could place this in the `tailwind.css` file as a global default? Would that be ok?